README.md

```diff
@@ -1452,7 +1452,21 @@ var event = await query.executeQuery();
 console.log(`>> Event: ${JSON.stringify(event)}`);
 ```
 
-
+# File Upload
+This API is used to upload a file and return the metadata that needs to be saved in delivery.
+```
+await client.fileUploader.upload(file);
+```
+will return
+```
+{
+  label: <file-name>,
+  md5: <md5-of-file-content>,
+  type: <file-type>,
```

Comment:
what are the possible values for type? Is it the extension? 
Make sure there's a comment so that it is not ambiguous